topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.md

```diff
@@ -0,0 +1,586 @@
+>    - In *"Track Group"*:
+>        - {% icon param-repeat %} *"Insert Track Group"*
+>            - *"Track Category"*: `Bakta`
+>            - In *"Annotation Track"*:
+>                - {% icon param-repeat %} *"Insert Annotation Track"*
+>                    - *"Track Type"*: `GFF/GFF3/BED Features`
+>                        - {% icon param-file %} *"GFF/GFF3/BED Track Data"*: `Annotation_and_sequences` (output of **Bakta** {% icon tool %})
+>                        - *"JBrowse Track Type [Advanced]"*: `Neat Canvas Features`
+>                        - *"Track Visibility"*: `On for new users`
+>        - {% icon param-repeat %} *"Insert Track Group"*
+>            - *"Track Category"*: `Plasmid sequences`
+>            - In *"Annotation Track"*:
+>                - {% icon param-repeat %} *"Insert Annotation Track"*
+>                    - *"Track Type"*: `GFF/GFF3/BED Features`
+>                        - {% icon param-file %} *"GFF/GFF3/BED Track Data"*: `PlasmidFinder GFF`
+>                        - *"JBrowse Track Type [Advanced]"*: `Neat Canvas Features`
+>                        - *"Track Visibility"*: `On for new users`
+>        - {% icon param-repeat %} *"Insert Track Group"*
+>            - *"Track Category"*: `IS elements`
+>            - In *"Annotation Track"*:
+>                - {% icon param-repeat %} *"Insert Annotation Track"*
+>                    - *"Track Type"*: `GFF/GFF3/BED Features`
+>                        - {% icon param-file %} *"GFF/GFF3/BED Track Data"*: GFF output of **ISEScan**
+>                        - *"JBrowse Track Type [Advanced]"*: `Neat Canvas Features`
+>                        - *"Track Visibility"*: `On for new users`
+>
+>        *If integrons are found as **IntegronFinder***
+>
+>        - {% icon param-repeat %} *"Insert Track Group"*
+>            - *"Track Category"*: `Integrons`
+>            - In *"Annotation Track"*:
+>                - {% icon param-repeat %} *"Insert Annotation Track"*
+>                    - *"Track Type"*: `GFF/GFF3/BED Features`
+>                        - {% icon param-file %} *"GFF/GFF3/BED Track Data"*: `IntegronFinder GFF`
+>                        - *"JBrowse Track Type [Advanced]"*: `Neat Canvas Features`
+>                        - *"Track Visibility"*: `On for new users`
+>
+> 3. View the output of JBrowse
+>
+{: .hands_on}
+
+In the output of the JBrowse you can view the genes, IS, plasmid, etc on the contigs. With the search tools you can easily find genes of interest. JBrowse can handle many inputs and can be very useful.
+
+If it takes too long to build the JBrowse instance, you can view an embedded one here. (**Warning**: feature name search will not work.)
+
+{% snippet topics/visualisation/faqs/visualizations_jbrowse.html datadir="data" loc="contig00019:1..30347" tracks="DNA,7e37ab7a5f0a1610de9d213b6a714666_0,dd555b269a8894688c3c7551d0d6a868_0,e1c36f2c4382e60836c6c4b4d8026fa5_0" %}
```

Comment:
Might need to say that it's ok to have the `Error reading from name store.` error

Reply:
Added a comment box with that